### ISSUE TITLE
DIPY CSD conversion

### ISSUE THREAD

**Issue by arokem** [PeerHerholz/bids_bep16_conv#4]
Same as #2, but with CSD instead of DTI.

**Comment by PeerHerholz** [comment]
Added `dipy_fit_CSD` function via this [commit](https://github.com/PeerHerholz/bids_bep16_conv/commit/071066647088757b2e9d96ecc1540ce46f0574a5). We now need to add the conversion function that adapts the output to BEP16-conform files.

**Comment by PeerHerholz** [comment]
Hi @arokem,

after changing the example dataset, the `CSD` analysis through `DIPY` is now working. When running it like this:

`dipy_fit_csd preproc_dwi.nii.gz preproc_dwi.bval preproc_dwi.bvec desc-brain_mask.nii.gz --out_dir outdir`

we get one file as a result: `peaks.pam5`. Is this expected/accurate or should the command be adapted?

**Comment by arokem** [comment]
Yes - this is not surprising, but will make the conversion a bit more tricky. That file format is an hdf5 format that stores information about the locations of peaks in the ODF and other metrics, such as spherical harmonic coefficients (hence "pam", for "peaks and metrics"). Some more information about this file is available here: https://github.com/dipy/dipy/blob/master/dipy/direction/peaks.py#L210
